frontend/src/metabase/visualizations/components/settings/ColumnItem/ColumnItem.tsx

```diff
@@ -41,60 +46,63 @@ const BaseColumnItem = ({
   onColorChange,
 }: ColumnItemProps) => {
   return (
-    <ColumnItemRoot
-      className={className}
-      role={role}
-      isDraggable={draggable}
-      onClick={onClick}
-      aria-label={role ? title : undefined}
-      data-testid={draggable ? `draggable-item-${title}` : null}
-      data-enabled={!!onRemove}
-    >
-      <ColumnItemContainer>
-        {draggable && <ColumnItemDragHandle name="grabber" />}
-        {onColorChange && color && (
-          <ColumnItemColorPicker
-            value={color}
-            onChange={onColorChange}
-            pillSize="small"
-          />
-        )}
-        <ColumnItemContent>
-          <ColumnItemSpan>
-            {icon && <Icon name={icon} />}
-            {title}
-          </ColumnItemSpan>
-          {onEdit && (
-            <ActionIcon
-              icon="ellipsis"
-              onClick={onEdit}
-              data-testid={`${title}-settings-button`}
-            />
-          )}
-          {onAdd && (
-            <ActionIcon
-              icon="add"
-              onClick={onAdd}
-              data-testid={`${title}-add-button`}
-            />
-          )}
-          {onRemove && (
-            <ActionIcon
-              icon="eye_outline"
-              onClick={onRemove}
-              data-testid={`${title}-hide-button`}
-            />
-          )}
-          {onEnable && (
-            <ActionIcon
-              icon="eye_crossed_out"
-              onClick={onEnable}
-              data-testid={`${title}-show-button`}
+    <HoverParent>
```

Comment:
`extra` could be anything, but by using `<HoverParent>` this component makes an assumption that `extra` will be used with `DelayGroup`.
WDYT about moving `<HoverParent>` to `SortableColumn` definition in `ChartSettingOrderedItems.tsx`, where the specific `extra` is passed?